diff --git a/Makefile b/Makefile
index 7bd1ed4..1be9359 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ PKG_NAME=$(NAME)-$(VERSION)
 PKG=$(PKG_DIR)/$(PKG_NAME).tar.gz
 SIG=$(PKG).asc
 
-PREFIX?=$(DESTDIR)/usr/local
+PREFIX?=/usr/local
 DOC_DIR=$(PREFIX)/share/doc/$(PKG_NAME)
 
 pkg:
@@ -59,10 +59,10 @@ rpm:
 	rpmbuild -ba rpm/chruby.spec
 
 install:
-	for dir in $(INSTALL_DIRS); do mkdir -p $(PREFIX)/$$dir; done
-	for file in $(INSTALL_FILES); do cp $$file $(PREFIX)/$$file; done
-	mkdir -p $(DOC_DIR)
-	cp -r $(DOC_FILES) $(DOC_DIR)/
+	for dir in $(INSTALL_DIRS); do mkdir -p $(DESTDIR)/$(PREFIX)/$$dir; done
+	for file in $(INSTALL_FILES); do cp $$file $(DESTDIR)/$(PREFIX)/$$file; done
+	mkdir -p $(DESTDIR)/$(DOC_DIR)
+	cp -r $(DOC_FILES) $(DESTDIR)/$(DOC_DIR)/
 
 uninstall:
 	for file in $(INSTALL_FILES); do rm -f $(PREFIX)/$$file; done
